tao_compiler/mlir/disc/transforms/disc_specialize_fusion_with_speculation.cc

```diff
@@ -471,11 +471,18 @@ struct DiscSpecializeFusionWithSpeculationPass
   // TODO(feiwen): add more kTileW=8/kTileH pairs by if/elseif/else
   void DoColReductionSpeculation(FusionOp fusion_op) {
     // We only do specialization fusion op on GPU a.t.m.
+    bool use_new = false;
+    static const char* env = getenv("NEW_COL");
```

Comment:
the env var looks informal?

Reply:
And it's the reason why it is WIP. I'm working on test to make sure the new strategy is constantly better so we can safely transit.